src/panels/lovelace/views/hui-view.ts

```diff
@@ -303,6 +355,19 @@ export class HUIView extends ReactiveElement {
     });
   }
 
+  private _createSections(config: LovelaceViewConfig): void {
+    if (!config || !config.sections || !Array.isArray(config.sections)) {
+      this._sections = [];
+      return;
+    }
+
+    this._sections = config.sections.map((sectionConfig, index) => {
+      const element = this.createSectionElement(sectionConfig);
+      element.index = index;
```

Comment:
Shouldn't we set `hass` here?

Reply:
It's already set inside `createSectionElement`